### PR TITLE
fix:Automatically unlock issue (#6770)

### LINKED ISSUE #6770: [OTHER] GitHub bot should unlock conversation once '🏁 status: ready for dev' label is added
### What would you like to share?

Currently, conversations in issues is locked while it is labelled `🚦 status: awaiting triage` and a maintainer needs to unlock it manually after label is changed to `🏁 status: ready for dev`. The conversation in #6658 is still locked even though it's labelled `🏁 status: ready for dev`.

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by nitheesh-daram**
## Fixes Issue

Resolves #6770 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots
When the `🚦 status: awaiting triage` label is added, it auto-locks the issue :
image
When  `🏁 status: ready for dev` is added, auto unlocks the issue :
image

## Note to reviewers

Replaced [repo-lockdown](https://github.com/marketplace/actions/repo-lockdown) with [label-actions](https://github.com/marketplace/actions/label-actions) as `label-actions` has both lock and unlock functionality and developed by the same developer.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7269"></a>

### CODE REVIEW

**Comment by nitheesh-daram** [comment]
Replaced [repo-lockdown](https://github.com/marketplace/actions/repo-lockdown) with [label-actions](https://github.com/marketplace/actions/label-actions) as label-actions has both lock and unlock functionality and developed by the same developer.

Here two things happen
1. It will `lock` the issue when the `🚦 status: awaiting triage` label is added, irrespective of other labels.
2. It will `Unlock` the issue when the `🏁 status: ready for dev` label is added, irrespective of other labels.

File: .github/lock-unlock-issues.yml
Comment (nitheesh-daram) [inline .github/lock-unlock-issues.yml:10]: You are free to change the comment,GitHub actions display while unlocking the issue.

**Comment by nitheesh-daram** [comment]
Looks like there's a bug where unlocking the issue goes `unlocks -> locks again -> unlocks`, will raise an issue in the [label-actions](https://github.com/dessant/label-actions) repo.

**Comment by nitheesh-daram** [comment]
> Looks like there's a bug where unlocking the issue goes `unlocks -> locks again -> unlocks`, will raise an issue in the [label-actions](https://github.com/dessant/label-actions) repo.
> 
> Update : raised here [dessant/label-actions#23](https://github.com/dessant/label-actions/issues/23)

This seems to happen only when you have a comment while unlocking an issue (Like we have)

**Comment by eddiejaoude** [comment]
> > Looks like there's a bug where unlocking the issue goes `unlocks -> locks again -> unlocks`, will raise an issue in the [label-actions](https://github.com/dessant/label-actions) repo.
> > Update : raised here [dessant/label-actions#23](https://github.com/dessant/label-actions/issues/23)
> 
> This seems to happen only when you have a comment while unlocking an issue (Like we have)

Oh I see, is this a blocker for us to use it? Should we wait until it is fixed on the original repo?

**Comment by nitheesh-daram** [comment]
> Oh I see, is this a blocker for us to use it? Should we wait until it is fixed on the original repo?

I think we are 2 options:
1. if we want a comment while unlocking, we need to wait for the new release (it's already fixed here(https://github.com/dessant/label-actions/commit/8ebc35d98bf4a8c61d6066fe76c310200e89a7f6) in the code part, but the owner has yet to release the new version).
2. if we don't want a comment while unlocking, we are good to go!

I would personally prefer option 2 as adding an extra comment while unlocking makes the issue thread unnecessarily long.

**Comment by eddiejaoude** [comment]
We do often comment, so we might have to wait for the release

**Comment by nitheesh-daram** [comment]
> We do often comment, so we might have to wait for the release

Sure, Eddie, I'll watch the repo and make the changes when a new release comes out.


And also, let me know what you would like to have as a comment while unlocking !
I suggest making it informative, like linking to docs on commit style,pr creation, rather than normal comments.

**Comment by eddiejaoude** [comment]
Awesome! I will mark this as block for now, and we can unblock it when the fix is released 👍 

I will check the text also 👍

**Comment by nitheesh-daram** [comment]
> Awesome! I will mark this as block for now, and we can unblock it when the fix is released +1
> 
> I will check the text also +1

Hey Eddie, the developer released a new release, and it looks like we don't need any changes from our end; we are good to merge!

image

File: .github/lock-unlock-issues.yml
Comment (nitheesh-daram) [inline .github/lock-unlock-issues.yml:10]: We need to change this to something informative.

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 , I will merge to a temporary branch and have a play

**Pull request merged**
